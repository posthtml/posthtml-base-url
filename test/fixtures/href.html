<a href="index.html">Go home</a>
<area shape="" coords="" href="docs/html" alt="">
<base href="docs/html">
<link rel="stylesheet" href="example.css">
