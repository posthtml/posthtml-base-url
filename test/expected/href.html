<a href="https://example.com/index.html">Go home</a>
<area shape="" coords="" href="https://example.com/docs/html" alt="">
<base href="https://example.com/docs/html">
<link rel="stylesheet" href="https://example.com/example.css">
